apps/workbench-client-testing-app/src/app/popup-opener-page/popup-opener-page.component.ts

```diff
@@ -92,7 +92,7 @@ export class PopupOpenerPageComponent implements AfterViewInit {
     this._coordinateAnchor$ = defer(() => this.form.get(ANCHOR)
       .valueChanges
       .pipe(
-        startWith(this.form.get(ANCHOR).value as object),
+        startWith(this.form.get(ANCHOR).value as Record<string, unknown>),
```

Comment:
I suggest removing this cast as not intuitive. I am aware that removing the cast will cause a deprecation warning, which, however, is a false-positive. What do you think?

Reply:
Introduced interface